plugins-compat-tester-model/src/main/java/org/jenkins/tools/test/model/PluginCompatReport.java

```diff
@@ -83,8 +83,9 @@ public void save(File reportPath) throws IOException {
         }
 
         // Writing to a temporary report file ...
-        File tempReportPath = new File(reportPath.getAbsolutePath()+".tmp");
-        try (Writer out = new FileWriter(tempReportPath)) {
+        File tempReportPath = new File(reportPath.getAbsolutePath() + ".tmp");
+        try (Writer out =
+                Files.newBufferedWriter(tempReportPath.toPath(), Charset.defaultCharset())) {
```

Comment:
There is an explicit encoding set in the line below. Should it use that one?

Reply:
Maybe, but in any case that is a pre-existing issue that wasn't introduced by my PR.